classes/Combination.php

```diff
@@ -81,6 +83,7 @@ class CombinationCore extends ObjectModel
             'ean13' => array('type' => self::TYPE_STRING, 'validate' => 'isEan13', 'size' => 13),
             'isbn' => array('type' => self::TYPE_STRING, 'validate' => 'isIsbn', 'size' => 32),
             'upc' => array('type' => self::TYPE_STRING, 'validate' => 'isUpc', 'size' => 12),
+            'mpn' => array('type' => self::TYPE_STRING, 'validate' => 'isMpn', 'size' => 32),
```

Comment:
Watch out, I know that SAP is using a 40 as max lenght.
```suggestion
            'mpn' => array('type' => self::TYPE_STRING, 'validate' => 'isMpn', 'size' => 40),
```